kge/job/entity_ranking.py

```diff
@@ -13,6 +13,11 @@ class EntityRankingJob(EvaluationJob):
 
     def __init__(self, config: Config, dataset: Dataset, parent_job, model):
         super().__init__(config, dataset, parent_job, model)
+        self.config.check(
+            "eval.entity_ranking.tie_handling",
```

Comment:
Why not: eval.tie_handling. It makes sense for all ranking tasks, even the ones we still plan to implement

Reply:
Yes that also makes sense. So dropping the key 'entity_ranking' completely for now.